endpoint-cors.php

```diff
@@ -68,6 +72,10 @@
     // and other POST requests (all requests are sent to the same endpoint in this example).
     // This condition is not needed if you don't require a callback on upload success.
     if (isset($_REQUEST["success"])) {
+        //If Region is set in uploadSuccess params then override the default us-east-1
+        $s3Region = (isset($_REQUEST["region"]) ? $_REQUEST["region"] : 'us-east-1');
```

Comment:
This scope will not work for delete operation. Delete operation will trigger error, because s3Region in delete operation scope will not be defined.